p/api/.htaccess

```diff
@@ -1,9 +1,14 @@
-<IfModule mod_setenvif.c>
-	SetEnvIfNoCase "^Authorization$" "(.*)" HTTP_AUTHORIZATION=$1
-</IfModule>
-<IfModule !mod_setenvif.c>
-	<IfModule mod_rewrite.c>
-		RewriteEngine on
-		RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
-	</IfModule>
-</IfModule>
+<IfVersion >= 2.4.13>
+    CGIPassAuth on
```

Comment:
This might generate an error 500 if users do not have the `AllowOverride` rights needed (`AuthConfig`, which is less commonly granted than `FileInfo`)